js/addToTask.js

```diff
@@ -0,0 +1,67 @@
+import {
+  CREATE_TASK_BUTTON,
+  CONTENT,
+  UL_LIST,
+  INPUT_TASK,
+  INPUT,
+  ADD_TASK,
+  TRASH_INPUT,
+  TODAY,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+window.taskList = [];
+window.id = 0;
```

Comment:
Don't directly insert objects into the window object. This beats the purpose of modular code and pollutes the global namespace. Please use local variable to set the `taskList` and pass the variable around when needed.

Reply:
okay